src/BaGet/appsettings.json

```diff
@@ -18,7 +18,7 @@
   },
 
   "Mirror": {
-    "Enabled": false,
+    "Enabled": true,
```

Comment:
Consider reverting this commit, see #300

Reply:
Thanks for reporting this, I've reverted it here https://github.com/loic-sharma/BaGet/commit/79ab701ab16b12f3ee951e7d43ffc90a85ad3f6a